staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_whitebox_test.go

```diff
@@ -2418,13 +2418,13 @@ func TestWatchStreamSeparation(t *testing.T) {
 			}
 			// Wait before sending watch progress request to avoid https://github.com/etcd-io/etcd/issues/17507
 			// TODO(https://github.com/etcd-io/etcd/issues/17507): Remove sleep when etcd is upgraded to version with fix.
-			time.Sleep(time.Second)
+			time.Sleep(100 * time.Millisecond)
 			err = cacher.storage.RequestWatchProgress(metadata.NewOutgoingContext(context.Background(), contextMetadata))
 			if err != nil {
 				t.Fatal(err)
 			}
 			// Give time for bookmark to arrive
-			time.Sleep(time.Second)
+			time.Sleep(100 * time.Millisecond)
```

Comment:
why do we need to sleep at all if the next thing we do is call `waitForEtcdBookmark()` which blocks until the event is received?

Reply:
but... waitForEtcdBookmark() will wait until we receive it, right? why do we need to sleep?

Reply:
not sure if this is related but `100ms` wasn't enough for https://github.com/kubernetes/kubernetes/pull/123926

Reply:
Right, increased to 200ms